xml/hardening.xml

```diff
@@ -4044,17 +4067,24 @@ RhostsRSAAuthentication no</screen>
       and data segments as well.
      </para>
      <para>
-      Another protection is called No-Execute (NX) or Data Execution
-      Prevention (DEP) which is enabled by default on the &productname;
-      kernel for the &x86; and &x86-64; architecture.
+      Executable space protection prevents the execution of memory space
+      which is not intended for execution. Linux makes use of the
+      <emphasis>No eXecute bit</emphasis>. This is enabled by default on
+      the &productname; kernel for the &x86; and &x86-64; architecture.
+      The usage of the NX bit has to be supported by each single
+      program. You can check if your system supports the NX bit:
      </para>
+<screen>&prompt.user;dmesg | grep '[NX|DX]*protection'
+[    0.000000] NX (Execute Disable) protection: active</screen>
+     <para>If NX is disabled, check your BIOS or UEFI for a setting that
+      enables it and make sure that it is supported by your CPU.</para>
      <para>
       Furthermore, since version 12, &productname; prevents leaking of
       internal kernel addresses to make kernel exploits harder by setting
       the <literal>kptr_restrict</literal>:
      </para>
-<screen># cat /proc/sys/kernel/kptr_restrict
-   1</screen>
+<screen>&prompt.user; cat /proc/sys/kernel/kptr_restrict
```

Comment:
Two issues:

* Remove the space between `&prompt.user;` and `cat`.
* Wrap `cat` with `<command>`.